app/controllers/api/v1/sessions_controller.rb

```diff
@@ -41,20 +41,29 @@ def session_params
       end
 
       def sign_in(user)
+        user.generate_session_token!(extended_session: session_params[:extend_session])
+
         # Creates an extended_session cookie if extend_session is selected in sign in form.
         if session_params[:extend_session]
           cookies.encrypted[:_extended_session] = {
             value: {
-              user_id: user.id
+              user_id: user.id,
+              session_token: user.session_token,
+              session_expiry: user.session_expiry
             },
             expires: 7.days,
-            httponly: true
+            httponly: true,
+            secure: true
```

Comment:
Using the secure attribute could mean that GL3 in a production environment will only be accessed through HTTPS.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#restrict_access_to_cookies

Reply:
This is intended

Reply:
It will be by default when we set up `force_ssl=true`

Reply:
Gotcha,